### ISSUE TITLE
Labels are too small when launching the viewer in 2D mode

### ISSUE THREAD

**Issue by aschier** [libigl/libigl#2204]
#### Describe the bug
Labels (Extra, Vertex, Face, etc.) are smaller than usual when the viewer is started in 2D mode:

	viewer.core(1).set_rotation_type(igl::opengl::ViewerCore::RotationType::ROTATION_TYPE_NO_ROTATION);
	viewer.launch();

Switching to Trackball or 2 axes makes the labels larger again. Afterward one can switch back to 2D and they keep the size.

In addition, using `viewer.data().label_size` to resize the labels results in rendering errors like overlapping letters. The default size (i.e. the larger size when using trackball view) is also a bit too small to be easy to read and has aliasing artifacts.

A screenshot of the smaller (2D perspective initialization) font size in actual size on a monitor with 2048x1152 resolution:

![A way to small font](https://github.com/libigl/libigl/assets/42997548/6680071d-7594-45c7-8054-69fa62233747)

A screenshot of the larger font size in actual size on a monitor with 2048x1152 resolution:

![A font with aliasing artifacts](https://github.com/libigl/libigl/assets/42997548/4567010e-6f5b-4fc8-8acd-4368f47c890f)

#### Platform

- [?] Windows
- [?] macOS
- [x] Linux

**Comment by alecjacobson** [comment]
I'm not planning on fixing this. Would be happy to accept a PR though.

**Comment by alecjacobson** [comment]
I wonder if this merged PR actually broke things. https://github.com/libigl/libigl/pull/1876

**Comment by alecjacobson** [comment]
nah... I don't think so. That's just introducing *data.label_size which is initialized to 1...
